### PR TITLE
Docs: use Ubuntu Pro in airgapped environment

### PR DESCRIPTION
**Pull request by pandrey2003**
## Proposed Commit Message
Adding documentation on how to use Ubuntu Pro in an airgapped environment.

## Checklist
 - [ ] I have updated or added any unit tests accordingly
 - [ ] I have updated or added any integration tests accordingly
 - [X] I have updated or added any documentation accordingly

## Does this PR require extra reviews?
 - [X] Yes
 - [ ] No
 
 Pro client: Lucas Moura (@lucasmoura).
 Product Management: Lech Sandeski (@lechsandecki).
 Field Engineering: Calvin Hartwell (@CalvinHartwell).
 Documentation: Sally Makin (@s-makin).

### CODE REVIEW

**Review by s-makin — APPROVED** [review]

File: docs/howtoguides/pro_in_airgapped.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+	deb https://bearer:[ESM_INFRA_RESOURCE_TOKEN]@esm.ubuntu.com/infra/ubuntu/ jammy-infra-updates main
+	deb https://bearer:[ESM_INFRA_RESOURCE_TOKEN]@esm.ubuntu.com/infra/ubuntu/ jammy-infra-security main
+
+	clean http://archive.ubuntu.com/ubuntu
+	```
+4. Run `apt-mirror`: \
+`sudo -u apt-mirror apt-mirror`
+5. Serve the `esm-infra` mirror on port `9090` (will be `[ESM_INFRA_MIRROR_URL]` in the next section): \
+`nohup sh -c 'python3 -m http.server --directory /var/spool/apt-mirror/mirror/esm.ubuntu.com/infra/ 9090 > /dev/null 2>&1 &'`
```
Comment (lucasmoura) [inline docs/howtoguides/pro_in_airgapped.md:76]: I think we can change this command to just:
`python3 -m http.server --directory /var/spool/apt-mirror/mirror/esm.ubuntu.com/infra/ 9090`

File: docs/howtoguides/pro_in_airgapped.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+```
+Run `ua-airgapped` for the new config in non-airgapped environment: `cat 02-overridden.yml | ua-airgapped > 02-server-ready.yml`
+
+`02-server-ready.yml` is our final configuration to run the server.
+
+## Running the server and final tweaks
+Run the following commands in the airgapped environment:
+1. Run the server: `contracts-airgapped --input=./02-server-ready.yml`
+2. Change the `contract_url` setting in `uaclient.conf` to point to the server: `http://1.2.3.4:8484`. Here `1.2.3.4` is the IP address of the airgapped machine.
+3. Run `sudo pro refresh` to check everything works fine.
```
Comment (lucasmoura) [inline docs/howtoguides/pro_in_airgapped.md:99]: I think we can ask the user to also run `pro attach` to see that the services are indeed enabled

File: docs/howtoguides/pro_in_airgapped.md (hunk 1 above)
Comment (pandrey2003) [reply docs/howtoguides/pro_in_airgapped.md]: Done.

File: docs/howtoguides/pro_in_airgapped.md (hunk 2 above)
Comment (pandrey2003) [reply docs/howtoguides/pro_in_airgapped.md:99]: Done.

**Review by lucasmoura — APPROVED** [review]

**Pull request merged**